sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/OffHeapColumnVector.java

```diff
@@ -73,12 +75,12 @@ public OffHeapColumnVector(int capacity, DataType type) {
     reset();
   }
 
-  @Override
+  @VisibleForTesting
   public long valuesNativeAddress() {
     return data;
   }
 
-  @Override
+  @VisibleForTesting
```

Comment:
Actually, this is not being called by any test.

Reply:
they are called in benchmark